specification/assets/gherkin/evaluation_v2.feature

```diff
@@ -0,0 +1,603 @@
+        Then the resolved details value should be "<resolved_value>"
+        And the reason should be "STATIC"
+
+        @booleans
+        Examples: Boolean evaluations
+            | key               | type    | default | resolved_value |
+            | boolean-zero-flag | Boolean | true    | false          |
+
+        @strings
+        Examples: String evaluations
+            | key              | type   | default | resolved_value |
+            | string-zero-flag | String | hi      |                |
+
+        @numbers
+        Examples: Number evaluations
+            | key               | type    | default | resolved_value |
+            | integer-zero-flag | Integer | 1       | 0              |
+            | float-zero-flag   | Float   | 0.1     | 0.0            |
+
+        @objects
+        Examples: Object evaluations
+            | key              | type   | default | resolved_value                                                                     |
+            | object-zero-flag | Object | {\"a\": 1} | {}                                                                                 |
+
+  # Spec 1.4.7: Testing TARGETING_MATCH reason with evaluation context
+  # Testing: dynamic context paradigm with targeting rules
+    @targeting @spec-1.4.7
+    Scenario Outline: Resolves zero value with targeting
+        Given a <type>-flag with key "<key>" and a default value "<default>"
+        And a context containing a key "email", with type "String" and with value "ballmer@macrosoft.com"
+        When the flag was evaluated with details
+        Then the resolved details value should be "<resolved_value>"
+        And the reason should be "TARGETING_MATCH"
+
+        @booleans
+        Examples: Boolean evaluations
+            | key                        | type    | default | resolved_value |
+            | boolean-targeted-zero-flag | Boolean | true    | false          |
+
+        @strings
+        Examples: String evaluations
+            | key                       | type   | default | resolved_value |
+            | string-targeted-zero-flag | String | hi      |                |
+
+        @numbers
+        Examples: Number evaluations
+            | key                        | type    | default | resolved_value |
+            | integer-targeted-zero-flag | Integer | 1       | 0              |
+            | float-targeted-zero-flag   | Float   | 0.1     | 0.0            |
+
+        @objects
+        Examples: Object evaluations
+            | key                       | type   | default    | resolved_value |
+            | object-targeted-zero-flag | Object | {\"a\": 1} | {}             |
+
+  # Spec 1.4.7: Testing DEFAULT reason when targeting doesn't match
+  # Testing: fallback to default value when context doesn't match targeting rules
+    @targeting @spec-1.4.7
+    Scenario Outline: Resolves zero value with targeting using default
+        Given a <type>-flag with key "<key>" and a default value "<default>"
```

Comment:
```suggestion
        Given a <type>-flag with key "<key>" and a fallback value "<default>"
```

For the test suite, lets try to use "fallback" for these cases to reduce confusion.